ui/raidboss/data/04-sb/dungeon/st_mocianne_hard.js

```diff
@@ -16,6 +16,7 @@ export default {
         text: {
           en: 'Be On A Platform',
           de: 'BSei auf einer Plattform',
+          fr: 'Soyez sur la plateforme A',
```

Comment:
```suggestion
          fr: 'Soyez sur une plateforme',
```

Parce que le "A" ici est un article indéfini, pas la lettre "A".

By the way, maybe the "A" in en text should be lower case?

Reply:
Oh Thank you.